google-map.html

```diff
@@ -431,7 +441,7 @@
         return;
       }
       this._mutationObserver = new MutationObserver( this._updateMarkers.bind(this));
-      this._mutationObserver.observe(this, {
+      this._mutationObserver.observe(this.$.selector, {
```

Comment:
You're right, there is one test case which fails if I run the tests using chromes native shadow dom implementation. It seems like the observer doesn't recognize the the changes inside the `<content>` tag. I tried putting the observer onto the `<content>` tag itself but it didn't work. What would be the best way to solve this? Any idea why this works in shady dom but not in shadow dom?